### ISSUE TITLE
Failed to load 'td_json_client_create': Symbol not found: node_register_module_v115

### ISSUE THREAD

**Issue by david-benes** [Bannerets/tdl#144]
I am trying to use tdl in apline linux with node 20. I am able to compile tdl successfully, however when I create the tdl client in node.js, it fails with:
`Failed to load 'td_json_client_create': Symbol not found: node_register_module_v115`

(v115 is an ABI version for node 20 as per https://nodejs.org/en/download/releases)

To demonstrate the issue, I created a minimal repository.
https://github.com/Jollor/tdl-alpine-build

Dockerfile.base contains build instructions.
index.js is creating tdl client.

When the container is started, the error appears.

```
workers-telegram-tdl  | Error: Failed to load 'td_json_client_create': Symbol not found: node_register_module_v115
workers-telegram-tdl  |     at loadAddon (/var/apps/workers/node_modules/tdl/dist/addon.js:52:9)
workers-telegram-tdl  |     at init (/var/apps/workers/node_modules/tdl/dist/index.js:72:38)
workers-telegram-tdl  |     at Object.createClient (/var/apps/workers/node_modules/tdl/dist/index.js:94:5)
workers-telegram-tdl  |     at Object.<anonymous> (/var/apps/workers/index.js:2:20)
workers-telegram-tdl  |     at Module._compile (node:internal/modules/cjs/loader:1241:14)
workers-telegram-tdl  |     at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
workers-telegram-tdl  |     at Module.load (node:internal/modules/cjs/loader:1091:32)
workers-telegram-tdl  |     at Module._load (node:internal/modules/cjs/loader:938:12)
workers-telegram-tdl  |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
workers-telegram-tdl  |     at node:internal/main/run_main_module:23:47
```

**Comment by Bannerets** [comment]
Thanks for reporting, this should be fixed in f32d94d9262b4b761de0f21e9d6ac391f722ca14. I released `tdl@7.3.2` with the fix.

**Comment by david-benes** [comment]
Wow, thanks a lot @Bannerets for the quick resolution. I can confirm that `tdl@7.3.2` resolves the issue.
